example/ios/StripeSdkExample.xcodeproj/project.pbxproj

```diff
@@ -383,6 +383,7 @@
 				COPY_PHASE_STRIP = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				ENABLE_TESTABILITY = YES;
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "arm64 ";
```

Comment:
Do we need this? I think it will break users on M1 MBPs.

Reply:
It's included in the upgrade guide for v0.66, but looks like then gets removed again for 0.67- https://react-native-community.github.io/upgrade-helper/?from=0.66.4&to=0.67.2 (which is still very new so not upgrading just yet)

Builds fine on my non-M1 with & without these additions, so I'm fine removing this